src/lib/ngsi/SubscriptionId.cpp

```diff
@@ -141,6 +141,40 @@ void SubscriptionId::present(const std::string& indent)
 
 /* ****************************************************************************
 *
+* SubscriptionId::toJson -
+*/
+std::string SubscriptionId::toJson(RequestType container, bool comma)
+{
+  std::string xString = string;
```

Comment:
We really choose a really bad name to store id string in Subscriptions.h... :)

NTC (just a bit surprised of my/our self :)

Reply:
This was probably me, smells KZ ... :-)
I don't find it that bad though :-D

NTC

Reply:
Just for fun... blame shows this:

![imagen](https://user-images.githubusercontent.com/1534240/32456734-e89b11a6-c326-11e7-8b26-cf6fedcbeffe.png)

Very ancient code, probably from first commit in public repo... The origin of this variable is lost in History :D

Reply:
99% sure it's mine :-)